.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,6 +18,9 @@ the problem you're facing:
    specific commit?
  - ...
 
+Note: If you are unsure whether your issue is a bug in the Zephyr codebase or if
+it is related to the misuse of Zephyr services, please create a Discussion topic.
```

Comment:
Should we mention discord as well? I think it's much more likely to hit another user there, especially since the channels have topics.

Reply:
I though about it but finally dropped it. I think we should try to push people to GH Discussions, which is more handy than Discord as it allows to provide more information and context to help understand where the issue lies

Reply:
Fair enough. The other thing I thought about is that GH Discussions and Issues, are indexed and show up in searches, while Discord conversations are probably all going to get lost and forgotten.